src/utilities/manifestStabilityUtils.ts

```diff
@@ -182,6 +244,33 @@ function isPodReady(podStatus: any): boolean {
    return allContainersAreReady
 }
 
+export function getContainerErrors(podStatus: any): string {
+   const errors: string[] = []
+   const collectErrors = (containers: any[], label: string) => {
+      containers?.forEach(({name, ready, state}) => {
+         if (!ready) {
```

Comment:
as this condition bypasses the remaining execution, i would recommend an early return here
` if (ready) return`

or 

```ts
if (ready){
  return
}
```